routers/web/user/setting/account.go

```diff
@@ -271,7 +275,7 @@ func loadAccountData(ctx *context.Context) {
 		user_model.EmailAddress
 		CanBePrimary bool
 	}
-	pendingActivation := ctx.Cache.IsExist("MailResendLimit_" + ctx.User.LowerName)
+	pendingActivation := setting.CacheService.Enabled && ctx.Cache.IsExist("MailResendLimit_"+ctx.User.LowerName)
```

Comment:
Uh, maybe we need to instead use a database query here when caching is disabled? because otherwise this will always be false, even if this is actually true.